impl/src/main/java/io/kroxylicious/testing/kafka/common/KafkaClusterConfig.java

```diff
@@ -230,7 +230,7 @@ public Stream<ConfigHolder> getBrokerConfigs(Supplier<KafkaEndpoints> endPointCo
                 catch (GeneralSecurityException | IOException e) {
                     throw new RuntimeException(e);
                 }
-                server.put(BrokerSecurityConfigs.SSL_CLIENT_AUTH_CONFIG, "required");
+                server.put(BrokerSecurityConfigs.SSL_CLIENT_AUTH_CONFIG, "requested");
```

Comment:
Is this what you meant in #20 @k-wall? My interpretation wasn't that is was a complaint about the broker config, but that when we got a client for an SSL broker it was automatically and always already configured for SSL, rather than get client being configured for plain by default and getting an SSL client being opt-in.

Reply:
AFAIK the client is always configured for PLAIN by default if you don't say in the test that you want to use TLS. That's why we need tests configured with `@Tls` (if we use the templates) annotation.

Is there any use case missing like having the broker configured with SSL and the client using no security at all? In that case, does it make sense? I read a lot of articles about kafka and no one recommends to use a plain client against an SSL broker (that's why I set it by default as `required`)